packages/grid/x-data-grid/src/models/params/gridCellParams.ts

```diff
@@ -106,6 +106,10 @@ export interface GridValueGetterParams<V = any, R = any>
    * @deprecated Use the `apiRef` returned by `useGridApiContext` or `useGridApiRef` (only available in `@mui/x-data-grid-pro`)
    */
   api: any;
+  /**
+   * The default value for the cell that the `valueGetter` is overriding.
```

Comment:
To override `GridCellParams['value']` description for `valueGetter`, since it doesn't make sense in that case (see https://github.com/mui/mui-x/issues/5823#issuecomment-1220669459)